### ISSUE TITLE
Selection of non-italic font leads to italic font loaded

### ISSUE THREAD

**Issue by brunoczim** [DraqueT/PolyGlot#887]
UPDATE: 

Sometimes on Windows 10, ALL FONTS selected other than the initial fonts will automatically render as italic. 
Notes: "Windows 10, and this happens if I switch to any font except “Serif” (and yes, this includes Charis SIL)."

On completion, inform reddit user bbrk24

ORIGINAL TEXT:

Suppose we have IPA /ˈne.ħaɑ/, we get this rendering:
![image](https://user-images.githubusercontent.com/18701450/80646013-56fb3600-8a42-11ea-844e-8b73c118a6f2.png)

Note that both "a" sounds rendered very similar, but they are different and this is confusing. This is due to the usage of italics.

**Comment by DraqueT** [comment]
Thank you for the heads up here! Just to make sure that I'm seeing this in the right place, are you looking at the print to PDF section? I'm having a bit of trouble finding where in the program that it's doing the italics. 

I'll ded change this to regular text, though. Your point is a good one.

**Comment by brunoczim** [comment]
Sorry, I forgot to say that this is in the lexicon, in a word's definition.

**Comment by brunoczim** [comment]
Oh, also happens in IPA Conversion Tool

**Comment by DraqueT** [comment]
I have been having a lot of difficulty tracking this down. Is it possible that you're using a local font that itself looks italic? This is set under the Language Preferences window. Just looking for other possibilities here, as this is not happening on my end. 

Also, are you on Windows? I want to make sure that I'm testing in the same environment that you are.

**Comment by brunoczim** [comment]
I am on Linux. Where is that window?

**Comment by DraqueT** [comment]
On the left side of the main menu, it is the Lang Properties button.

**Comment by brunoczim** [comment]
You mean local font? It still is in italics.

**Comment by DraqueT** [comment]
Ok, I think that is the problem there. It seems like the local font that you've chosen is itself in italics. This is would explain why I wasn't able to reproduce it. If you choose a new font which is not italic, it should correct your issue.

**Comment by brunoczim** [comment]
The problem is I did not select an italic font. I selected a regular Noto Sans. In fact, changing the font won't change what is rendered.

**Comment by DraqueT** [comment]
Which specific noto sans font did you select? There might be some sort of incompatibility with the font itself that I will have to test for.

**Comment by DraqueT** [comment]
As a heads up, this seems like an issue that can be resolved without editing code, so I'm going to close this ticket. If you would still like some help getting this set up properly, please toss me an email over at draquemail@gmail.com and I can give a hand. :3

**Comment by DraqueT** [comment]
REOPENING this one! Looks like there are underlying issues that can pop up occasionally which I had overlooked. ^^

Notes: "Windows 10, and this happens if I switch to any font except “Serif” (and yes, this includes Charis SIL)."

**Comment by bobby-bolton** [comment]
@DraqueT how's the progress on this one? The default font won't accept some of my characters, and when switching to Calibri, everything's italicized and a mess.

**Comment by DraqueT** [comment]
Heyo, sorry to leave this one for so long (day job has become wild lately). I've been having trouble recreating it over on my end. I know that it's happening for some users, so I know it's not just some fluke on your side, but haven't yet been able to make the bug appear in my own tests yet. I can't remember, did you already send me the language file that you're seeing the problem in? If not, could you send it over, and if so, what was it named again?

**Comment by DraqueT** [comment]
Closing this one now. The long and the short of this is that the OS font chooser in Java is fundamentally broken. I have replaced this functionality so that users are now heavily pushed toward selecting font files manually to import. The option to use the OS font chooser still exists, but pops a warning dialogue before letting the user continue.

This issue does not appear when importing fonts manually.
